### PR TITLE
Remove add

### PR DESCRIPTION
**Pull request by Alzubair98**
all the requirements are done 

1- add Add function
2- add remove function
3- add edit function

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,15 @@
 # Project Name
```
Comment (zainsadaqat) [inline README.md:1]: Please add the Project Name here 👍🏻

File: README.md [hunk 2]
```diff
@@ -1,18 +1,15 @@
...
 
-to do list
-
-# Description
-
-the second project in the second week of the second module.
-a simple but useful to do list
+set up the webpack
```
Comment (zainsadaqat) [inline README.md:3]: Please add some description about this project. What it is all about 👍🏻

File: README.md [hunk 3]
```diff
@@ -1,18 +1,15 @@
...
 
 ## Built With
 
 - Major languages
 - Frameworks
 - Technologies used
```
Comment (zainsadaqat) [inline README.md:9]: Please mention the technologies which are being used to build this project 👍🏻

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @Alzubair98  😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

- [x] It's not updating the local storage. As you can see in the screenshot, I've selected the second item and it should turn the flag to true. and it should not store the already deleted item in the local storage as it's doing by storing the item 2.
Screenshot 2022-07-07 at 11 03 08 PM


_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (Alzubair98) [reply README.md:9]: hello @zainsadaqat  can you please watch this short video 
i explained everything in it

https://www.loom.com/share/2bba079a20164f4d9d7b17494c58fd20

**Review by EricMbouwe — APPROVED** [review]
Hi @Alzubair98, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Add changes requested
- No linters errors
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**